bin/installMaven

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# download and unzip Maven to ~/maven directory
+# and overwrite the mvn command to run this Maven version instead of Travis' standard maven version (3.5.2)
+
+set -euo pipefail
+
+echo "Setup Maven"
+mkdir -p ~/maven
+pushd ~/maven > /dev/null
+if [ ! -d "apache-maven-$1" ]; then
+  echo "Download Maven $1"
+  wget --quiet --continue  "https://archive.apache.org/dist/maven/maven-3/$1/binaries/apache-maven-$1-bin.zip" 
+  unzip apache-maven-$1-bin.zip -d ~/maven> /dev/null
+fi
+popd > /dev/null
+export M2_HOME="~/maven/apache-maven-$1"
```

Comment:
`~` doesn't expand inside double-quotes. You could write `export M2_HOME=~/maven/apache-maven-"$1"` but actually you don't need double-quotes in variable assignments, so `export M2_HOME=~/maven/apache-maven-$1` will work and is perfectly safe.